src/main/webapp/status.xhtml

```diff
@@ -6,12 +6,12 @@
       xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
       xmlns:f="http://xmlns.jcp.org/jsf/core"
       xmlns:p="http://primefaces.org/ui" xmlns:c="http://java.sun.com/jsp/jstl/core">
-<h:head>
-    <title>Service Status -- Aberfitness</title>
-</h:head>
-<h:body>
     <ui:composition template="./aberfitnessTemplate.xhtml">
 
+        <ui:define name="page_head">
+            <title>Service Status - Aberfitness</title>
```

Comment:
Should be a capital F in "fitness" - sorry!!

Reply:
Done